### ISSUE TITLE
InputEvent.device: it's not documented what value corresponds to "All devices".

### ISSUE THREAD

**Issue by kleonc** [godotengine/godot-docs#4413]
**Your Godot version:**
Any.

**Issue description:**
It seems that value of `ALL_DEVICES` constant:
https://github.com/godotengine/godot/blob/b9a2787bd164602e866ecac1527020e0e1c34da8/core/input/input_map.cpp#L39
is not documented anywhere.
I suggest exposing that constant in the documentation and adding some reference to the exposed constant in the description of `InputEvent.device`.

**URL to the documentation page (if already existing):**
- `InputMap`: [3.2](https://docs.godotengine.org/en/3.2/classes/class_inputmap.html), [stable](https://docs.godotengine.org/en/stable/classes/class_inputmap.html), [latest](https://docs.godotengine.org/en/latest/classes/class_inputmap.html),
- `InputEvent.device`: [3.2](https://docs.godotengine.org/en/3.2/classes/class_inputevent.html#class-inputevent-property-device), [stable](https://docs.godotengine.org/en/stable/classes/class_inputevent.html#class-inputevent-property-device), [latest](https://docs.godotengine.org/en/latest/classes/class_inputevent.html#class-inputevent-property-device).

**Comment by Calinou** [comment]
Also, should this constant also be exposed to scripting?

**Comment by kleonc** [comment]
@Calinou Yeah, I guess I meant that too. It probably should be analogous to for example [TileMap.INVALID_CELL](https://docs.godotengine.org/en/3.2/classes/class_tilemap.html#constants) (off topic: which seems to don't have the best description since it not always have to be "returned").

**Comment by tetrapod00** [comment]
Still valid in 4.x. See also https://github.com/godotengine/godot/pull/97707 and linked PRs, which changed this value (and was then reverted due to side effects).
